network-api/networkapi/wagtailpages/templatetags/bg_nav_tags.py

```diff
@@ -11,7 +11,8 @@ def check_active_category(current_category, target_category):
     # make sure to compare the linguistic originals.
     current_category = getattr(current_category, 'original', current_category)
     target_category = getattr(target_category, 'original', target_category)
-    return 'active' if current_category == target_category else ''
+    match = current_category == target_category or current_category.parent == target_category
+    return 'active' if match else ''
```

Comment:
Turns out this could break on `None` categories, so let's make this just a touch more robust:
```suggestion
    match = current_category == target_category
    if hasattr(current_category, 'parent'):
        match = match or current_category.parent == target_category
    return 'active' if match else ''
```